src/analysis/schwartz_space.lean

```diff
@@ -403,6 +405,52 @@ begin
   rwa [pow_zero, one_mul] at this,
 end
 
+/-- The supremum of all seminorms `schwartz_map.seminorm 𝕜 k' n'` with `k' ≤ k` and `n' ≤ n`. -/
+@[protected]
+def seminorm_sup (k n : ℕ) : seminorm 𝕜 𝓢(E, F) :=
+((finset.range (k + 1)) ×ˢ (finset.range (n + 1))).sup (λ n, seminorm 𝕜 n.1 n.2)
```

Comment:
```suggestion
(finset.Iic (k, n)).sup (λ n, seminorm 𝕜 n.1 n.2)
```

Reply:
With that it might be even better to remove the definition, what do you think?

Reply:
Your call! I agree it is now quite short indeed.